lib/widgets/lightbox.dart

```diff
@@ -295,8 +354,8 @@ class _ImageLightboxPageState extends State<_ImageLightboxPage> {
       elevation: elevation,
       child: Row(children: [
         _CopyLinkButton(url: widget.src),
+        if (Platform.isAndroid) _DownloadImageButton(url: widget.src),
         // TODO(#43): Share image
-        // TODO(#42): Download image
```

Comment:
nit: put the `_DownloadImageButton` after the share-image TODO, matching the order of the TODOs